prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+   var fullName = 'Frank Aguirre';
+   console.log('value of name ' + fullName);
+
+   var numberOfPets = '2';
+   console.log('number of pets ' + numberOfPets);
```

Comment:
Hi Frank, be sure to leave the quotes off of your number so that the JavaScript engine does not interpret it as a string.

Reply:
just sent updated pull.